.github/CONTRIBUTING.md

```diff
@@ -309,6 +343,7 @@ Math operators like +, -, /, *, etc are up in the air, just choose which version
 #### Use
 * Bitwise AND - '&'
 	* Should be written as ```bitfield & bitflag``` NEVER ```bitflag & bitfield```, both are valid, but the latter is confusing and nonstandard.
+	* Feel free to use one of the helpful bitflag operation macros instead (CHECK_BITFIELD and similar).
```

Comment:
I still really hate these things

Reply:
At least we are not using this another codebase does: `#define LAZYLEN(L) length(L)`
:p

Reply:
Should I remove it or keep it?